api/namespace/applications/routes.py

```diff
@@ -108,7 +108,10 @@ def put(self):
             application_id, section_name, section_dict
         )
 
-        return updated_section, 201
+        if updated_section:
+            return updated_section, 201
```

Comment:
⛏️ Should this be "make_response"?

Reply:
The updated_section is json serialisable so rest-x should be able to handle it (as well as make_response)